### PR TITLE
Contour - Update docs for new contour version

### PR DESCRIPTION
**Pull request by jsanin-vmw**
## Which other branches do you want a technical writer to cherry-pick this PR to (if any)?

No need to cherry pick

These changes are for the upcoming release TAP 1.6

### CODE REVIEW

File: about-package-profiles.hbs.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ a predefined group of packages.
 
   Contour is an ingress controller for Kubernetes that supports dynamic configuration updates and multi-team ingress
   delegation. It provides the control plane for the Envoy edge and service proxy. For more information about Contour, see
-  the [Contour documentation](https://projectcontour.io/docs/v1.22.0/).
+  the [Contour documentation](https://projectcontour.io/docs/v1.24/).
```
Comment (KauzClay) [inline about-package-profiles.hbs.md:101]: I believe this link is broken, you can probably remove the v and make it work

File: contour/configuring-cipher-suites-and-tls-version.hbs.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ Contour provides configuration options for TLS version and Cipher Suites.
 Rather than directly exposed through a top level key in the pacakge, 
 they fall into the category of advanced Contour configurations by using the `contour.configFileContents` key. 
 For more information about these configuration options, 
-see [Contour documentation](https://projectcontour.io/docs/v1.23.1/configuration/). 
+see [Contour documentation](https://projectcontour.io/docs/v1.24/configuration/).
```
Comment (KauzClay) [inline contour/configuring-cipher-suites-and-tls-version.hbs.md:9]: I think this link is broken too, try https://projectcontour.io/docs/1.24/configuration/

File: contour/install.hbs.md [hunk 3]
```diff
@@ -92,40 +91,79 @@ To install Contour from the Tanzu Application Platform package repository withou
...
+          certificates.duration                                     8760h                     string   How long the certificates should be valid for.                                    
+          certificates.renewBefore                                  360h                      string   How long before expiration the certificates should be renewed.                    
+          contour.pspNames                                          vmware-system-restricted  string   Pod security policy names to apply to Contour.                                    
+          contour.replicas                                          2                         integer  How many Contour pod replicas to have.                                            
+          contour.resources.contour.requests.memory                                           string   Memory request to apply to the contour container.                                 
+          contour.resources.contour.requests.cpu                                              string   CPU request to apply to the contour container.
```
Comment (KauzClay) [inline contour/install.hbs.md:120]: I wonder if we should make a note in the release notes about being able to set resource limits and requests for the Contour and Envoy pods now?

Comment (jsanin-vmw) [reply contour/install.hbs.md:120]: release notes added. Let me know what you think

Comment (KauzClay) [reply contour/install.hbs.md:120]: nice, looks good to me

**Comment by KauzClay** [comment]
/approve

trying to indicate that this has the stamp of approval from the component team side 👍

**Pull request merged**